marimba_ur5_moveit_config/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1.3)
+project(marimbabot_ur5_moveit_config)
```

Comment:
rename the folder from "marimba_ur5_moveit_config" to "marimbabot_ur5_moveit_config" for consistency ?

Reply:
also I guess that it is best practice that project and folder name match